SpatialGDK/Documentation/content/non-unreal-server-worker-types.md

```diff
@@ -171,54 +180,64 @@ You could add a new entity with the given component in your Unreal project code
 
 ```
 UCLASS()
-class SPATIALGDK_API USessionEntitySnapshotGeneration : USnapshotGenerationTemplate
+class UTestEntitySnapshotGeneration : public USnapshotGenerationTemplate
 {
-    GENERATED_BODY()
+	GENERATED_BODY()
```

Comment:
This and the newly added code below is using tabs while the snippet above looks like it's using spaces. Not sure if this could affect rendering?

Reply:
Mmm I just copied both from VS so not sure how this happened. Looks identical in improbadoc preview though.